sentry-spring/src/main/java/io/sentry/spring/SentryRequestFilter.java

```diff
@@ -1,25 +1,27 @@
-package io.sentry.spring.boot;
+package io.sentry.spring;
 
 import com.jakewharton.nopen.annotation.Open;
 import io.sentry.core.IHub;
 import io.sentry.core.SentryOptions;
+import io.sentry.core.util.Objects;
 import java.io.IOException;
 import javax.servlet.FilterChain;
 import javax.servlet.ServletException;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import org.jetbrains.annotations.NotNull;
+import org.springframework.core.Ordered;
 import org.springframework.web.filter.OncePerRequestFilter;
 
 /** Pushes new {@link io.sentry.core.Scope} on each incoming HTTP request. */
 @Open
-public class SentryRequestFilter extends OncePerRequestFilter {
+public class SentryRequestFilter extends OncePerRequestFilter implements Ordered {
   private final @NotNull IHub hub;
   private final @NotNull SentryOptions options;
 
   public SentryRequestFilter(final @NotNull IHub hub, final @NotNull SentryOptions options) {
-    this.hub = hub;
-    this.options = options;
+    this.hub = Objects.requireNonNull(hub, "hub is required");
+    this.options = Objects.requireNonNull(options, "options are required");
```

Comment:
the reason to not use the original JDK method `Objects.requireNonNull` is because its not available on older Android APIs, but it'd be available in this package as its non-Android.
I'll leave to you to keep this one (io.sentry.core.util.Objects) or to use the original `Objects`, this is just a JFYI comment.

Reply:
My bad, I intended to use the one from `Sentry`

Reply:
Ah sorry I misread. I think if we just use everywhere the same guards it's less confusing. Spring has it's own built in guards (which are actually more reasonable than the ones from JDK) as they don't throw `NullPointerException` but `IllegalArgumentException`.

Reply:
cool, let's keep our own then.